internal/primitive/transform/action/strings/replace_between_positions_test.go

```diff
@@ -44,38 +44,38 @@ func TestReplaceBetweenPositionsAction(t *testing.T) {
 		a, err := runtime.NewAction([]interface{}{funcName, "$.test", 100, 8, "Dan"})
 		So(err, ShouldBeNil)
 		e := cetest.MinEvent()
-		e.SetExtension("test","Hello Joe, Vanus is an amazing technology")
+		e.SetExtension("test","Start position must be less than the length of the string")
```

Comment:
```suggest
e.SetExtension("test", "Hello Joe, Vanus is an amazing technology")
```

why did you change this line code? The string is the same in the code on lines 47 and 53 because the function execute failed to return an error,  so the string doesn't be changed

Reply:
Okay sir, I've fixed it across the file